src/deploy/NVA_build/standalone_deploy_nsfs.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# Assumes that a postgresql is already running
+
+# This deployment script just automates the steps in the doc/standalone.md file
+
+function execute() {
+    if [[ -z "${CEPH_TEST_LOGS_DIR}" ]]; then
+        $1 &
+    else
+        $1 2>&1 | tee "${CEPH_TEST_LOGS_DIR}/${2}" &
+    fi
+}
+
+function main() {
+
+    # Add accounts to run ceph tests
+    execute "node src/cmd/manage_nsfs account add --config_root ./standalone/config_root --name cephalt --email ceph.alt@noobaa.com --new_buckets_path ./standalone/nsfs_root --access_key abcd --secret_key abcd" nsfs_cephalt.log
+    execute "node src/cmd/manage_nsfs account add --config_root ./standalone/config_root --name cephtenant --email ceph.tenant@noobaa.com --new_buckets_path ./standalone/nsfs_root --access_key efgh --secret_key efgh" nsfs_cephtenant.log
+
+    # Start nsfs server
+    execute "node src/cmd/nsfs --config_root ./standalone/config_root" nsfs.log
```

Comment:
save the name of the file in a constant (log file name).

Reply:
sure